vividus-extension-selenium/src/test/java/org/vividus/steps/ui/validation/BaseValidationsTests.java

```diff
@@ -165,6 +167,21 @@ void testAssertIfElementDoesNotExistWithLocator()
         assertFalse(searchParameters.isWaitForElement());
     }
 
+    @Test
+    void testAssertElementDoesNotExistWithLocator()
+    {
+        spy = Mockito.spy(baseValidations);
```

Comment:
Any reason to have a spy as a class field? org.mockito.Spy you can also consider this annotation